apps/src/lab2/projects/ProjectManagerFactory.ts

```diff
@@ -39,13 +41,18 @@ export default class ProjectManagerFactory {
    */
   static async getProjectManagerForLevel(
     projectManagerStorageType: ProjectManagerStorageType,
+    userId: string,
```

Comment:
In the same vein as @thomasoniii's comment, should `userId` be optional here like `scriptId`?

Reply:
Good call; now done.